packages/devtools/lib/src/timeline/event_details.dart

```diff
@@ -128,6 +129,7 @@ class EventDetails extends CoreElement {
   }
 
   Future<void> update(FrameFlameChartItem item) async {
+    gaSelect(gaTimeline, gaTimelineFlame);
```

Comment:
nit: can you put this in `onSelectedFrameFlameChartItem.listen` instead? Seems more fitting to have it where we are listening for the selection. That is in timeline.dart line 183.

Reply:
Done and it returns the duration in the event_value as well.